### DISCUSSION TITLE
already migrating to dove for personal project?

### DISCUSSION THREAD

**Discussion by chrisLovesCode** [feathersjs/feathers#2403] [General]
Hey there,
i'm not an big feathers expert, is just startet something last year and like to continue it.
since my last use of feathers in 2020/12, it seems feathers 5 is on its way and i'm thinking about updating my WIP project. Is feathers 5 more or less usable in terms of core features? (oauth and postrgress support for instance).
I learned that dove will get better typescript support(yeah!) and add custom methods....any other cool main features? 

thx for advice

**Comment by bwgjoseph** [comment]
You can take a look at [v5-docs](https://dove.docs.feathersjs.com/) and the [migrating](https://dove.docs.feathersjs.com/guides/migrating.html) section to see what changed!
